tests/storage/linstor/test_linstor_sr.py

```diff
@@ -65,7 +65,7 @@ def test_start_and_shutdown_VM(self, vm_on_linstor_sr):
         vm = vm_on_linstor_sr
         vm.start()
         vm.wait_for_os_booted()
-        vm.shutdown(verify=True)
+        vm.shutdown(verify=True, force_if_fails=True)
```

Comment:
Why is `force_if_fails` added here? We might want to check that the VM normal shutdown can happen in the test.
Was it something that was discussed that I missed?

Reply:
Right, the test purpose is getting diluted. Will revert this.